contracts/Certificate.sol

```diff
@@ -103,7 +123,7 @@ contract Certificate is
    * @param purchasingTokenAddress The address of the token used to purchase the certificate.
    * @param priceMultiple The number of purchasing tokens required to buy one NRT.
    */
-  event ReceiveRemovalBatch(
+  event CreateCertificate(
```

Comment:
Don't have a strong opinion on renaming this but we should be careful since @jaycenhorton has some changes in his branch that would break CI (and make it difficult to resolve) if this was merged.
Not saying we shouldn't do it, just make sure to coordinate with him on this.

Reply:
Made a new ticket to capture our intent to rename and will revert this rename for now

Reply:
we'll want to rename on a branch where we're going to actual do a fresh deploy from